service/templates/settings.html

```diff
@@ -140,7 +145,7 @@ <h3 style="padding-left:15px;"><i class="fa fa-globe"></i> User Settings</h3>
                 </div>
               </div>
             </div>
-            <div class="right text-right"><button type="submit" ng-click="updateUserSettings(new_tz, use_spruce_options, spruce_feedback)" class="btn btn-primary">Save</button></div>
+            <div class="right text-right"><button type="submit" ng-click="updateUserSettings(new_tz, new_region, use_spruce_options, spruce_feedback)" class="btn btn-primary">Save</button></div>
```

Comment:
This function call has too many params now

Reply:
True... should new_tz also be changed to work like new_region? Is there a reason it's included in the function call?

Reply:
oh yea it should, missed that one